network/netblock_test.go

```diff
@@ -40,6 +39,9 @@ func TestNetblock(t *testing.T) {
 
 			require.Equal(t, tt.cidr, netblock.Cidr.String())
 			require.Equal(t, tt.netType, netblock.Type)
+
+			fmt.Println("netblock asset type is", netblock.AssetType())
```

Comment:
let's drop this Println please

Reply:
I'm happy to remove it but maybe a conversation about unit test output is in order. It would be helpful to me, anyway.